### ISSUE TITLE
Unhandled exceptions

### ISSUE THREAD

**Issue by aogclosure** [domialex/Sidekick#414]
2020-06-23 00:37:47.129 -05:00 [INF] Keybind Triggered - Open Search
2020-06-23 00:37:48.315 -05:00 [FTL] Unhandled exception in application root
System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (Keyword "Alt" is not valid.)
 ---> System.ArgumentException: Keyword "Alt" is not valid.
   at System.Windows.Forms.SendKeys.ParseKeys(String keys, IntPtr hwnd)
   at System.Windows.Forms.SendKeys.Send(String keys, Control control, Boolean wait)
   at System.Windows.Forms.SendKeys.SendWait(String keys)
   at Sidekick.Natives.NativeKeyboard.SendInput(String input)
   at Sidekick.Natives.KeybindEvents.<>c__DisplayClass58_0.<<SendInputIf>b__0>d.MoveNext()
   --- End of inner exception stack trace ---

2020-06-23 00:45:40.185 -05:00 [INF] Keybind Triggered - Open Search
2020-06-23 00:56:53.717 -05:00 [FTL] Unhandled exception in application root
System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (Keyword "Alt" is not valid.)
 ---> System.ArgumentException: Keyword "Alt" is not valid.
   at System.Windows.Forms.SendKeys.ParseKeys(String keys, IntPtr hwnd)
   at System.Windows.Forms.SendKeys.Send(String keys, Control control, Boolean wait)
   at Sidekick.Natives.NativeKeyboard.SendInput(String input)
   at Sidekick.Natives.KeybindEvents.<>c__DisplayClass58_0.<<SendInputIf>b__0>d.MoveNext()
   --- End of inner exception stack trace ---
2020-06-23 00:56:53.812 -05:00 [FTL] Unhandled exception in application root
System.Threading.Tasks.TaskCanceledException: A task was canceled.
   at System.Windows.Threading.DispatcherOperation.Wait(TimeSpan timeout)
   at System.Windows.Threading.Dispatcher.InvokeImpl(DispatcherOperation operation, CancellationToken cancellationToken, TimeSpan timeout)
   at System.Windows.Threading.Dispatcher.Invoke(Action callback, DispatcherPriority priority, CancellationToken cancellationToken, TimeSpan timeout)
   at System.Windows.Threading.Dispatcher.Invoke(Action callback)
   at Sidekick.App.LogUnhandledException(Exception ex)
   at Sidekick.App.<AttachErrorHandlers>b__18_2(Object s, UnobservedTaskExceptionEventArgs e)
   at System.Threading.Tasks.TaskScheduler.PublishUnobservedTaskException(Object sender, UnobservedTaskExceptionEventArgs ueea)
   at System.Threading.Tasks.TaskExceptionHolder.Finalize()
2020-06-23 00:57:05.957 -05:00 [INF] Changed active language support to English.
2020-06-23 00:57:06.179 -05:00 [DBG] 0% - Checking for Updates... UpdateManager
2020-06-23 00:57:06.764 -05:00 [DBG] 0% - Initializer - Start Reset KeybindEvents
2020-06-23 00:57:06.768 -05:00 [DBG] 0% - Initializer - End Reset KeybindEvents
2020-06-23 00:57:06.768 -05:00 [DBG] 0% - Initializer - Start Reset NativeKeyboard
2020-06-23 00:57:06.769 -05:00 [DBG] 0% - Initializer - End Reset NativeKeyboard
2020-06-23 00:57:06.770 -05:00 [DBG] 0% - Initializer - Start Before Init LeagueDataService
2020-06-23 00:57:06.773 -05:00 [INF] Fetching leagues started.
2020-06-23 00:57:06.997 -05:00 [INF] 4 leagues fetched.
2020-06-23 00:57:07.000 -05:00 [DBG] 7% - Initializer - End Before Init LeagueDataService
2020-06-23 00:57:07.002 -05:00 [DBG] 7% - Initializer - Start Init StatDataService
2020-06-23 00:57:07.331 -05:00 [DBG] 15% - Initializer - End Init StatDataService
2020-06-23 00:57:07.331 -05:00 [DBG] 15% - Initializer - Start Init ItemDataService
2020-06-23 00:57:07.388 -05:00 [DBG] 23% - Initializer - End Init ItemDataService
2020-06-23 00:57:07.388 -05:00 [DBG] 23% - Initializer - Start Init PseudoStatDataService
2020-06-23 00:57:07.392 -05:00 [INF] Pseudo stat service initialization started.
2020-06-23 00:57:07.409 -05:00 [INF] 9 attributes fetched.
2020-06-23 00:57:07.499 -05:00 [DBG] 30% - Initializer - End Init PseudoStatDataService
2020-06-23 00:57:07.499 -05:00 [DBG] 30% - Initializer - Start Init StaticDataService
2020-06-23 00:57:07.507 -05:00 [DBG] 38% - Initializer - End Init StaticDataService
2020-06-23 00:57:07.507 -05:00 [DBG] 38% - Initializer - Start Init UILanguageProvider
2020-06-23 00:57:07.507 -05:00 [DBG] 46% - Initializer - End Init UILanguageProvider
2020-06-23 00:57:07.508 -05:00 [DBG] 46% - Initializer - Start After Init ItemDataService
2020-06-23 00:57:07.509 -05:00 [DBG] 53% - Initializer - End After Init ItemDataService
2020-06-23 00:57:07.509 -05:00 [DBG] 53% - Initializer - Start After Init PoeNinjaClient
2020-06-23 00:57:07.509 -05:00 [DBG] 61% - Initializer - End After Init PoeNinjaClient
2020-06-23 00:57:07.509 -05:00 [DBG] 61% - Initializer - Start After Init PoeNinjaCache
2020-06-23 00:57:07.511 -05:00 [INF] Populating PoeNinja cache.
2020-06-23 00:57:07.772 -05:00 [INF] PoeNinja cache populated.
2020-06-23 00:57:07.772 -05:00 [DBG] 69% - Initializer - End After Init PoeNinjaCache
2020-06-23 00:57:07.772 -05:00 [DBG] 69% - Initializer - Start After Init ParserPatterns
2020-06-23 00:57:07.777 -05:00 [DBG] 76% - Initializer - End After Init ParserPatterns
2020-06-23 00:57:07.777 -05:00 [DBG] 76% - Initializer - Start After Init KeybindEvents
2020-06-23 00:57:07.778 -05:00 [DBG] 84% - Initializer - End After Init KeybindEvents
2020-06-23 00:57:07.778 -05:00 [DBG] 84% - Initializer - Start After Init NativeKeyboard
2020-06-23 00:57:07.778 -05:00 [DBG] 92% - Initializer - End After Init NativeKeyboard
2020-06-23 00:57:07.778 -05:00 [DBG] 92% - Initializer - Start After Init NativeProcess
2020-06-23 00:57:07.779 -05:00 [DBG] 100% - Initializer - End After Init NativeProcess
2020-06-23 00:57:09.801 -05:00 [INF] Permission Sufficient.
2020-06-23 00:57:11.380 -05:00 [INF] Keybind Triggered - Check Prices
2020-06-23 00:57:12.074 -05:00 [INF] Querying Trade API.
2020-06-23 00:57:12.241 -05:00 [INF] Fetching Trade API Listings from Query gkje2p6IQ.
2020-06-23 00:57:12.479 -05:00 [INF] Fetching Trade API Listings from Query gkje2p6IQ.
2020-06-23 00:57:17.324 -05:00 [INF] Querying Trade API.
2020-06-23 00:57:17.438 -05:00 [INF] Fetching Trade API Listings from Query Gl4JB6WIb.
2020-06-23 00:57:17.568 -05:00 [INF] Fetching Trade API Listings from Query Gl4JB6WIb.
2020-06-23 00:57:24.997 -05:00 [INF] Querying Trade API.
2020-06-23 00:57:25.137 -05:00 [INF] Fetching Trade API Listings from Query 3jzYK2Gc5.
2020-06-23 00:57:25.279 -05:00 [INF] Fetching Trade API Listings from Query 3jzYK2Gc5.
2020-06-23 00:57:30.975 -05:00 [INF] Querying Trade API.
2020-06-23 00:57:31.103 -05:00 [INF] Fetching Trade API Listings from Query LBRovn5Tn.
2020-06-23 00:57:38.575 -05:00 [INF] Querying Trade API.
2020-06-23 00:57:38.827 -05:00 [INF] Fetching Trade API Listings from Query q2w3849Cg.
2020-06-23 00:57:52.671 -05:00 [INF] Querying Trade API.
2020-06-23 00:57:52.794 -05:00 [INF] Fetching Trade API Listings from Query 0ndZ3ewcg.
2020-06-23 00:59:11.463 -05:00 [INF] Keybind Triggered - Go to Hideout
2020-06-23 00:59:11.621 -05:00 [INF] ChatService - Wrote '/hideout' in the chat

2020-06-23 01:11:41.421 -05:00 [INF] Keybind Triggered - Open Search
2020-06-23 01:12:06.893 -05:00 [FTL] Unhandled exception in application root
System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (Keyword "Alt" is not valid.)
 ---> System.ArgumentException: Keyword "Alt" is not valid.
   at System.Windows.Forms.SendKeys.ParseKeys(String keys, IntPtr hwnd)
   at System.Windows.Forms.SendKeys.Send(String keys, Control control, Boolean wait)
   at System.Windows.Forms.SendKeys.SendWait(String keys)
   at Sidekick.Natives.NativeKeyboard.SendInput(String input)
   at Sidekick.Natives.KeybindEvents.<>c__DisplayClass58_0.<<SendInputIf>b__0>d.MoveNext()
   --- End of inner exception stack trace ---
2020-06-23 01:16:57.549 -05:00 [INF] Changed active language support to English.
2020-06-23 01:16:57.727 -05:00 [DBG] 0% - Checking for Updates... UpdateManager
2020-06-23 01:16:58.294 -05:00 [DBG] 0% - Initializer - Start Reset KeybindEvents
2020-06-23 01:16:58.298 -05:00 [DBG] 0% - Initializer - End Reset KeybindEvents
2020-06-23 01:16:58.298 -05:00 [DBG] 0% - Initializer - Start Reset NativeKeyboard
2020-06-23 01:16:58.299 -05:00 [DBG] 0% - Initializer - End Reset NativeKeyboard
2020-06-23 01:16:58.300 -05:00 [DBG] 0% - Initializer - Start Before Init LeagueDataService
2020-06-23 01:16:58.302 -05:00 [INF] Fetching leagues started.
2020-06-23 01:16:58.516 -05:00 [INF] 4 leagues fetched.
2020-06-23 01:16:58.519 -05:00 [DBG] 7% - Initializer - End Before Init LeagueDataService
2020-06-23 01:16:58.520 -05:00 [DBG] 7% - Initializer - Start Init StatDataService
2020-06-23 01:16:58.826 -05:00 [DBG] 15% - Initializer - End Init StatDataService
2020-06-23 01:16:58.826 -05:00 [DBG] 15% - Initializer - Start Init ItemDataService
2020-06-23 01:16:58.882 -05:00 [DBG] 23% - Initializer - End Init ItemDataService
2020-06-23 01:16:58.882 -05:00 [DBG] 23% - Initializer - Start Init PseudoStatDataService
2020-06-23 01:16:58.886 -05:00 [INF] Pseudo stat service initialization started.
2020-06-23 01:16:58.903 -05:00 [INF] 9 attributes fetched.
2020-06-23 01:16:58.988 -05:00 [DBG] 30% - Initializer - End Init PseudoStatDataService
2020-06-23 01:16:58.989 -05:00 [DBG] 30% - Initializer - Start Init StaticDataService
2020-06-23 01:16:58.996 -05:00 [DBG] 38% - Initializer - End Init StaticDataService
2020-06-23 01:16:58.996 -05:00 [DBG] 38% - Initializer - Start Init UILanguageProvider
2020-06-23 01:16:58.997 -05:00 [DBG] 46% - Initializer - End Init UILanguageProvider
2020-06-23 01:16:58.998 -05:00 [DBG] 46% - Initializer - Start After Init ItemDataService
2020-06-23 01:16:58.999 -05:00 [DBG] 53% - Initializer - End After Init ItemDataService
2020-06-23 01:16:58.999 -05:00 [DBG] 53% - Initializer - Start After Init PoeNinjaClient
2020-06-23 01:16:58.999 -05:00 [DBG] 61% - Initializer - End After Init PoeNinjaClient
2020-06-23 01:16:58.999 -05:00 [DBG] 61% - Initializer - Start After Init PoeNinjaCache
2020-06-23 01:16:59.001 -05:00 [INF] Populating PoeNinja cache.
2020-06-23 01:16:59.257 -05:00 [INF] PoeNinja cache populated.
2020-06-23 01:16:59.258 -05:00 [DBG] 69% - Initializer - End After Init PoeNinjaCache
2020-06-23 01:16:59.258 -05:00 [DBG] 69% - Initializer - Start After Init ParserPatterns
2020-06-23 01:16:59.262 -05:00 [DBG] 76% - Initializer - End After Init ParserPatterns
2020-06-23 01:16:59.263 -05:00 [DBG] 76% - Initializer - Start After Init KeybindEvents
2020-06-23 01:16:59.263 -05:00 [DBG] 84% - Initializer - End After Init KeybindEvents
2020-06-23 01:16:59.263 -05:00 [DBG] 84% - Initializer - Start After Init NativeKeyboard
2020-06-23 01:16:59.263 -05:00 [DBG] 92% - Initializer - End After Init NativeKeyboard
2020-06-23 01:16:59.264 -05:00 [DBG] 92% - Initializer - Start After Init NativeProcess
2020-06-23 01:16:59.264 -05:00 [DBG] 100% - Initializer - End After Init NativeProcess
2020-06-23 01:17:29.374 -05:00 [INF] Permission Sufficient.
2020-06-23 01:17:35.759 -05:00 [INF] Keybind Triggered - Check Prices
2020-06-23 01:17:36.249 -05:00 [INF] Querying Trade API.
2020-06-23 01:17:36.365 -05:00 [INF] Fetching Trade API Listings from Query XLLzp8eIP.
2020-06-23 01:17:36.571 -05:00 [INF] Fetching Trade API Listings from Query XLLzp8eIP.
2020-06-23 01:17:39.230 -05:00 [INF] Keybind Triggered - Check Prices
2020-06-23 01:17:39.586 -05:00 [INF] Querying Trade API.
2020-06-23 01:17:39.711 -05:00 [INF] Fetching Trade API Listings from Query wrrE2qKUb.
2020-06-23 01:17:39.874 -05:00 [INF] Fetching Trade API Listings from Query wrrE2qKUb.
2020-06-23 01:17:43.804 -05:00 [INF] Keybind Triggered - Check Prices
2020-06-23 01:17:44.163 -05:00 [INF] Querying Exchange API.
2020-06-23 01:17:44.279 -05:00 [INF] Fetching Trade API Listings from Query MdDZQ7yiJ.
2020-06-23 01:17:44.431 -05:00 [INF] Fetching Trade API Listings from Query MdDZQ7yiJ.
2020-06-23 01:17:49.261 -05:00 [INF] Keybind Triggered - Check Prices
2020-06-23 01:17:49.481 -05:00 [ERR] Could not parse item.
System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Sidekick.Business.Apis.Poe.Parser.ItemSections.get_HeaderSection()
   at Sidekick.Business.Apis.Poe.Trade.Data.Items.ItemDataService.ParseItemData(ItemSections itemSections)
   at Sidekick.Business.Apis.Poe.Parser.ParserService.ParseItem(String itemText)
2020-06-23 01:17:52.661 -05:00 [INF] Keybind Triggered - Check Prices
2020-06-23 01:17:53.007 -05:00 [INF] Querying Exchange API.
2020-06-23 01:17:53.127 -05:00 [INF] Fetching Trade API Listings from Query MdDZQ7yiJ.
2020-06-23 01:17:53.269 -05:00 [INF] Fetching Trade API Listings from Query MdDZQ7yiJ.
2020-06-23 01:17:56.153 -05:00 [INF] Keybind Triggered - Check Prices
2020-06-23 01:17:56.371 -05:00 [ERR] Could not parse item.
System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Sidekick.Business.Apis.Poe.Parser.ItemSections.get_HeaderSection()
   at Sidekick.Business.Apis.Poe.Trade.Data.Items.ItemDataService.ParseItemData(ItemSections itemSections)
   at Sidekick.Business.Apis.Poe.Parser.ParserService.ParseItem(String itemText)
2020-06-23 01:18:00.386 -05:00 [INF] Keybind Triggered - Check Prices
2020-06-23 01:18:00.736 -05:00 [INF] Querying Exchange API.
2020-06-23 01:18:00.869 -05:00 [INF] Fetching Trade API Listings from Query MdDZQ7yiJ.
2020-06-23 01:18:01.029 -05:00 [INF] Fetching Trade API Listings from Query MdDZQ7yiJ.
2020-06-23 01:19:17.138 -05:00 [INF] Keybind Triggered - Check Prices
2020-06-23 01:19:17.579 -05:00 [INF] Querying Trade API.
2020-06-23 01:19:17.869 -05:00 [INF] Fetching Trade API Listings from Query WQ8RlgRsm.
2020-06-23 01:19:18.046 -05:00 [INF] Fetching Trade API Listings from Query WQ8RlgRsm.
2020-06-23 01:19:30.034 -05:00 [INF] Querying Trade API.
2020-06-23 01:19:30.171 -05:00 [INF] Fetching Trade API Listings from Query mZ6Y6zaS6.
2020-06-23 01:19:30.293 -05:00 [INF] Fetching Trade API Listings from Query mZ6Y6zaS6.
2020-06-23 01:19:35.763 -05:00 [INF] Querying Trade API.
2020-06-23 01:19:35.924 -05:00 [INF] Fetching Trade API Listings from Query lo6a6n6FV.
2020-06-23 01:19:36.087 -05:00 [INF] Fetching Trade API Listings from Query lo6a6n6FV.
2020-06-23 01:19:37.824 -05:00 [INF] Fetching Trade API Listings from Query lo6a6n6FV.
2020-06-23 01:19:44.186 -05:00 [INF] Querying Trade API.
2020-06-23 01:19:44.303 -05:00 [INF] Fetching Trade API Listings from Query Q3l9l6jcw.
2020-06-23 01:19:44.445 -05:00 [INF] Fetching Trade API Listings from Query Q3l9l6jcw.
2020-06-23 01:19:48.583 -05:00 [INF] Querying Trade API.
2020-06-23 01:19:48.690 -05:00 [INF] Fetching Trade API Listings from Query Gl4J7B3sb.
2020-06-23 01:19:48.838 -05:00 [INF] Fetching Trade API Listings from Query Gl4J7B3sb.
2020-06-23 01:20:34.234 -05:00 [INF] Keybind Triggered - Open Wiki
2020-06-23 01:20:40.020 -05:00 [INF] Keybind Triggered - Open Search
2020-06-23 01:20:40.229 -05:00 [FTL] Unhandled exception in application root
System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (Keyword "Alt" is not valid.)
 ---> System.ArgumentException: Keyword "Alt" is not valid.
   at System.Windows.Forms.SendKeys.ParseKeys(String keys, IntPtr hwnd)
   at System.Windows.Forms.SendKeys.Send(String keys, Control control, Boolean wait)
   at System.Windows.Forms.SendKeys.SendWait(String keys)
   at Sidekick.Natives.NativeKeyboard.SendInput(String input)
   at Sidekick.Natives.KeybindEvents.<>c__DisplayClass58_0.<<SendInputIf>b__0>d.MoveNext()
   --- End of inner exception stack trace ---

**Comment by domialex** [comment]
Duplicate of #413 probably, which is fixed.
